packager/mux/mux.go

```diff
@@ -0,0 +1,93 @@
+package mux
+
+import (
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/ampproject/amppackager/packager/certcache"
+	"github.com/ampproject/amppackager/packager/signer"
+	muxp "github.com/ampproject/amppackager/packager/mux/params"
+	"github.com/ampproject/amppackager/packager/util"
+	"github.com/ampproject/amppackager/packager/validitymap"
+)
+
+type mux struct {
+	certCache *certcache.CertCache
+	signer *signer.Signer
+	validityMap *validitymap.ValidityMap
+}
+
+// Implements the HTTP routing strategy used by the amppkg server. The main
```

Comment:
Maybe move all/most of these comments to package level.

Reply:
Done.